rollup.config.js

```diff
@@ -36,12 +37,6 @@ export default {
     resolve({
       mainFields: ['module', 'main', 'jsnext', 'browser'],
     }),
-    commonjs({
-      sourceMap: true,
-      namedExports: {
-        react: ['isValidElement'],
-      },
-    }),
```

Comment:
Can you explain why you have to change that in the context of this PR?

Reply:
`commonjs` plugin is only needed when you actually want to **bundle** a commonjs dependency - you don't really want that with `react` since you are leaving it as external (like you should), so this was actually a "dead"/redundant config

Reply:
Let's improve this!

Reply:
Hm, could you be more specific? maybe I've explained this poorly - so let me rephrase: the removed code that was not actually used and it wasn't doing anything here. And how this config now is defined is correct - from my point of view this change is "sound" and correct. There is no regression here so this can be merged "as is". Please let me know if this addresses your concerns - if not I can try to explain it further and in more detail.

Reply:
Ho! Sorry I just wanted to said _Let's change this!_ in the meaning of accepting your change with pleasure. Sorry for the wrong wording :smiling_face_with_tear: